spec/support/matchers/accessibility.rb

```diff
@@ -1,3 +1,27 @@
+RSpec::Matchers.define :have_valid_idrefs do
+  invalid_idref_messages = []
+
+  match do |page|
+    ['aria-describedby', 'aria-labelledby'].each do |idref_attribute|
+      page.all(:css, "[#{idref_attribute}]").each do |element|
+        page.find_by_id(element[idref_attribute]) # rubocop:disable Rails/DynamicFindBy
+      rescue Capybara::ElementNotFound
+        invalid_idref_messages << "[#{idref_attribute}=\"#{element[idref_attribute]}\"]"
+      end
+    end
+
+    invalid_idref_messages.blank?
+  end
+
+  failure_message do |page|
+    <<-STR.strip_heredoc
```

Comment:
I think we can do the same thing by switching to `<<~` from `<<-`
```suggestion
    <<~STR
```

Reply:
> I think we can do the same thing by switching to `<<~` from `<<-`

TIL! Thanks.